addon/components/bourbon-search-select-field.js

```diff
@@ -59,7 +59,12 @@ export default Component.extend(SelectMixin, {
   mouseDown() {
     this.set('activeOption', null);
     this.set('showDropdown', !this.get('showDropdown'));
-    this.set('inputValue', '');
+
+    if (this.get('showDropdown') === true) {
```

Comment:
if `showDropdown` is a boolean, this should probably just be `if (this.get('showDropdown')`, no need to compare with `true`

Reply:
lol...yeah.  i will update.  :)